arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -214,6 +214,7 @@ public CreateSection(Hub hub) {
             this, hub.currentFrame()); // To get the success bit of the CREATE(2)
 
     hub.romLex().callRomLex(frame);
+    // hub.transients().conflation().deploymentInfo().newDeploymentAt(createeAddress);
```

Comment:
?

Reply:
I took it out here as it's already done at `traceContextEnter` in `Hub.java` so this would be doing it twice

![image](https://github.com/user-attachments/assets/62c3783f-a28c-403d-87a0-b92eb0979514)

Reply:
OK maybe add a dic comment to explain it ?

Reply:
I removed it altogether (the comment.)